natlas-server/app/scope/scope_manager.py

```diff
@@ -1,21 +1,18 @@
-import ipaddress
 from netaddr import IPNetwork, IPAddress, IPSet
 from netaddr.core import AddrFormatError
-from .ipscanmanager import IPScanManager
+from .scan_manager import IPScanManager
 from datetime import datetime
-import os
 from flask import current_app
 
 LOGFILE = "logs/scopemanager.log"
 
 
 def log(message: str, printm: bool = False):
-    if not os.path.isdir("logs"):
-        os.makedirs("logs", exist_ok=True)
+    msg = f"{str(datetime.utcnow())} - {message}\n"
     with open(LOGFILE, "a") as f:
-        f.write(f"{str(datetime.utcnow())} - {message}\n")
+        f.write(msg)
     if printm:
-        print(f"{str(datetime.utcnow())} - {message}\n")
+        print(f"{msg}\n")
```

Comment:
Why aren't we delegating to an existing logging implementation?

Reply:
I'll tackle this one another time, I want to remove any and all disparate logging functions like this, use the actual logging library, and default to only logging to stdout so that we can stop making that silly logs directory